apply changes in the extensibility docs (#1851)
kyma-project/busola: docs/extensibility/README.md

## Overview

> Extensibility in Busola is a feature that allows you to create a separate dedicated user interface (UI) page for your CustomResourceDefinition (CRD). It allows you to add navigation nodes, on Cluster or Namespace level, and to configure your [UI display](display-section.md), for example, a resource list page, and details pages. You can also [create and edit forms](form-section.md).

Reviewer: ```suggestion
With Busola's extensibility feature, you can create a separate dedicated user interface (UI) page for your CustomResourceDefinition (CRD). It enables you to add navigation nodes, on cluster or Namespace level, and to configure your [UI display](display-section.md), for example, a resource list page, and details pages. You can also [create and edit forms](form-section.md).
```

Is "cluster" capitalized? 
Otherwise, just my usual allergy against "allows" when we're talking about stuff that never was forbidden.
